src/window/building_info.c

```diff
@@ -419,7 +417,7 @@ static void init(int grid_offset)
                     case FIGURE_FRIENDLY_ARROW:
                     case FIGURE_WATCHTOWER_ARCHER:
                         break;
-                    // intentional fallthrough
+                        // intentional fallthrough
```

Comment:
There doesn't seem to be an intentional  fall through here anymore, so this line may go